src/app/Traits/HasEvent.php

```diff
@@ -7,7 +7,7 @@
 
 trait HasEvent
```

Comment:
I think this trait is not really necessary, Adrian was suggesting using a local Resource if/when we'd want to create custom calendar from Models, rather than using the trait on the models